scripts/provisioning/configcmd.py

```diff
@@ -281,9 +281,10 @@ def configure_haproxy(self):
 
       # Create main config file for haproxy.
       S3HaproxyConfig(self.url).process()
+      # update the haproxy log rotate config file in /etc/logrotate.d/haproxy
+      self.find_and_replace("/etc/logrotate.d/haproxy", "/var/log/cortx", self.base_log_file_path)
```

Comment:
This needs to be in 'container init script'

Reply:
done. moved this to s3_start file